docs/getting-started/what-is-dotnet.md

```diff
@@ -133,18 +139,17 @@ The .NET Framework on the Windows desktop allows you to create console applicati
 
 - Console applications can be compiled to run cross-platform on Windows, the Macintosh, and Linux.
 
-- Console applications can be compiled natively. This combines the benefits of a managed application with the performance of a native C/C++ native application. 
+- Console applications can be compiled natively. This combines the benefits of a managed application with the performance of a native C/C++ application. 
```

Comment:
Native compilation: This combines the benefits of a managed app with the performance of a native C/C++ app.